lib/internal/test_runner/mock.js

```diff
@@ -131,8 +132,10 @@ class MockTracker {
     implementation = kDefaultFunction,
     options = kEmptyObject,
   ) {
-    validateObject(object, 'object');
     validateStringOrSymbol(methodName, 'methodName');
+    if (typeof object !== 'function') {
```

Comment:
should it be named something else if it’s an object or a function?

Reply:
I think I didn't get what you mean.

Reply:
I think the recommendation is to rename `object` to something that indicates that the value can also be a function. Maybe just call it `objectOrFunction`.

Reply:
done!